### PR TITLE
Refactor frontend to ES6

### PR DESCRIPTION
**Pull request by Karyum**
#136

### CODE REVIEW

**Comment by Karyum** [comment]
@finnhodgkin cheers merge conflict resolved

File: public/js/helpers/date.js [hunk 1]
```diff
@@ -1,13 +1,10 @@
-var toDateString = function (date) {
+const toDateString = date =>
   // + 1 on the month because month is zero-indexed
-  return (
-    date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate()
-  );
-};
+  `${date.getFullYear()}-${date.getMonth() + 1}- ${date.getDate()}`;
```
Comment (finnhodgkin) [inline public/js/helpers/date.js]: Is there supposed to be an extra space before the final getDate? The old version didn't have that

File: public/js/validate_form.js [hunk 2]
```diff
@@ -4,77 +4,75 @@ import axios from 'axios';
...
 
-  function validatedLanguage(input, messageBox, lang) {
+  const validatedLanguage = (input, messageBox, lang) => {
     window.location.href = '#eventName';
     document.querySelector('#eventName').children[input].style =
       'border: 0.1rem red solid';
-    var messageBoxElement = document.querySelector(messageBox);
-    messageBoxElement.innerHTML = 'Should contain ' + lang + ' characters only';
+    const messageBoxElement = document.querySelector(messageBox);
+    messageBoxElement.innerHTML = `Should contain ${lang} characters only`;
```
Comment (finnhodgkin) [inline public/js/validate_form.js]: And here.

File: public/js/validate_form.js [hunk 3]
```diff
@@ -4,77 +4,75 @@ import axios from 'axios';
...
     if (childInputs) {
-      var inputParent = document.querySelector(input).children;
+      const inputParent = document.querySelector(input).children;
       inputParent[0].style = 'border: 0.1rem gray solid';
       inputParent[1].style = 'border: 0.1rem gray solid';
     } else {
       var container = document.querySelector(input);
       container.style = 'border: 0.1rem gray solid';
     }
     document.querySelector(messageBox).innerHTML = '';
```
Comment (finnhodgkin) [inline public/js/validate_form.js]: Any reason for this to be innerHTML and not textContent?

Comment (Karyum) [reply public/js/validate_form.js]: good shout didn't pay attention to this

Comment (finnhodgkin) [reply public/js/validate_form.js]: Good shout but not fixing?

**Comment by finnhodgkin** [comment]
Still hardly ever linking to issues in commits :man_facepalming:

**Pull request merged**